docs/cow-protocol/reference/core/auctions/competition_rules.md

```diff
@@ -1,20 +1,30 @@
 ---
-id: social-consensus
+id: competition-rules
 sidebar_position: 2
 ---
 
-# Social consensus rules
+# The current rules of the solver competition
 
-Social consensus rules are not enforced by the smart contract. However, by voting on them in a CoW improvement proposal (CIP), CoW DAO has decided that these rules should be followed to ensure a healthy competition. 
+All solvers participating in the solver competition must abide by certain rules. In this section, we outline all these rules, which are naturally split into two classes: (1) the ones enforced explicitly by the smart contract and the offchain protocol infrastructure, and (2) the so-called social consensus rules.
+
+## Rules enforced by the smart contract and/or the autopilot
```

Comment:
I'd likely suggest breaking this down more atomically such that it was:

1. Smart contract
2. Protocol
3. Social consensus (which still seems like it fits under the Protocol bracket, as this is enforced by the autopilot, or is there a distinction here?)

Reply:
Yeah good point, will do that.

Social consensus are not actually enforced anywhere. The way they are used is the following: "you should follow these rules although they are not really hard rules and are not enforced on a per auction basis. However we monitor settlements and if we catch you, you will be in trouble and might get slashed...."